.github/workflows/ci.yml

```diff
@@ -58,10 +58,10 @@ jobs:
           cache: 'maven'
       - name: Configure Problem Matchers
         run: echo "::add-matcher::.github/problem-matcher.json"
-      - name: Maven Checks
+      - name: Run Maven checks on all commits
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $RETRY $MAVEN clean verify -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
+          git rebase master -x "$RETRY $MAVEN clean verify -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'"
```

Comment:
Can you show an example of failure for a non-head commit? Would it be easy enough to identify which commit failed?

Also, I don't think we want to run all checks on all commits except the head. We only want to make sure other commits compile.

Reply:
```
Error:  Failed to execute goal io.trino:trino-maven-plugin:11:generate-service-descriptor (default-generate-service-descriptor) on project trino-hive-hadoop2: Execution default-generate-service-descriptor of goal io.trino:trino-maven-plugin:11:generate-service-descriptor failed: A required class was missing while executing io.trino:trino-maven-plugin:11:generate-service-descriptor: io/trino/plugin/hive/HiveConnectorFactory
Error:  -----------------------------------------------------
Error:  realm =    extension>io.trino:trino-maven-plugin:11
Error:  strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
Error:  urls[0] = file:/home/runner/.m2/repository/io/trino/trino-maven-plugin/11/trino-maven-plugin-11.jar
Error:  urls[1] = file:/home/runner/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.17/plexus-utils-3.0.17.jar
Error:  Number of foreign imports: 1
Error:  import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
Error:  
Error:  -----------------------------------------------------: io.trino.plugin.hive.HiveConnectorFactory
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :trino-hive-hadoop2
interactive rebase in progress; onto daa858da30
Last commands done (6 commands done):
   pick 3fc5c79e20 Intentionally break a plugin
   exec ./mvnw package -B --quiet --strict-checksums --no-snapshot-updates --no-transfer-progress -T C1 -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -Dair.check.skip-all=true -P gib -Dgib.referenceBranch=refs/remotes/origin/master -pl '!:trino-server-rpm' || (git status && false)
  (see more in file .git/rebase-merge/done)
No commands remaining.
You are currently editing a commit during a rebase.
  (use "git commit --amend" to amend the current commit)
  (use "git rebase --continue" once you are satisfied with your changes)

nothing to commit, working tree clean
warning: execution failed: ./mvnw package -B --quiet --strict-checksums --no-snapshot-updates --no-transfer-progress -T C1 -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -Dair.check.skip-all=true -P gib -Dgib.referenceBranch=refs/remotes/origin/master -pl '!:trino-server-rpm' || (git status && false)
You can fix the problem, and then run

  git rebase --continue

```